features/admin/statistics.feature

```diff
@@ -22,7 +22,7 @@ Feature: Statistics
         And 2 more new customers have paid 2 orders placed for total of "$5,000.37"
         When I view statistics
         Then I should see 5 new customers
-        And I should see 6 new orders
+        And I should see 6 paid orders
```

Comment:
Becouse it's not actually a new order, just a paid one, the same like in our dashboard
![Screenshot 2024-01-15 at 12 42 37](https://github.com/Sylius/Sylius/assets/53942444/11f59872-efa9-450d-a7d9-80724aa9da8b)